.gitignore

```diff
@@ -37,3 +37,4 @@ node_modules
 package.json
 package-lock.json
 temp
+wandb/
```

Comment:
extra?

Reply:
oh my bad, it was from the cifar branch that I added to ignore for conveniences